### DISCUSSION TITLE
nopcommerce integration with orchard CMS

### DISCUSSION THREAD

**Discussion by tripurarisingh1** [OrchardCMS/OrchardCore#13778] [General]
We are using Orchard CMS 1.6.0 and we wanted to integration of nopcommerce inside orchard, Can you please help me on this ?

**Comment by hishamco** [comment]
You can create a custom module to integrate with nopcommerce APIs, or you could try [Orchard.Commerce](https://github.com/OrchardCMS/OrchardCore.Commerce)

FYI there will be a session today in Orchard Core Harvest, I'm sure it will be informative for you https://orchardcore.net/harvest
